Flexa-Agents/large-deposit-withdrawal/README.md

```diff
@@ -0,0 +1,28 @@
+# Large Deposit Withdrawal Agent
+
+## Description
+
+This agent detects deposits and withdrawals in Flexa staking contract with an amount that exceeds 1M USD.
+
+> Amp token price is obtained using chainlink data feeds.
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- FLEXA-1
+  - Fired when a transaction deposits/ withdraws a large amount to Flexa Staking contract.
+  - Severity is always set to "Info".
+  - Type is always set to "Info".
+  - Metadata contains:
+    - `supplier`: Address of the user executing the deposit/withdrawal.
+    - `amount`: Amount of tokens transfered.
+
+## Test Data
+
+There's no current transaction including a Deposit or Withdrawal with an amount greater than 1M USD. To verify the agent behavior, you can change the threshold to a lower amount and test with the following transactions:
```

Comment:
Maybe specifying an amount that could be used for the lower threshold to test these transactions with would be good.

Reply:
I couldn't do that because the USD amount will change based on the token price, so the value can no longer be working in the future

Reply:
But you are using the price relative to the block when that transactions occurred right? If that is the case Roberto suggestion will work.

Reply:
Ah yes you're right, after the last change this became possible. Thanks for pointing that out.